### PR TITLE
Fix inconsistent msmt pages

### PR DESCRIPTION
**Pull request by sarathms**
This PR contains proposed fixes to the issues discussed recently.
* Restructuring the code in `WebConnectivity.js` measurement page component
* Injecting the message from `scores.msg` for WhatsApp measurement pages

### CODE REVIEW

File: components/measurement/nettests/WebConnectivity.js [hunk 1]
```diff
@@ -287,10 +283,10 @@ const WebConnectivityDetails = ({
...
   }
 
-  let status = 'default',
-    // XXX: `reason` will soon be used in Hero
-    reason = null,
-    summaryText = ''
+  let status = 'default'
+  let summaryText = ''
+
+  const reason = messages[`blockingReason.${blocking}`] && intl.formatMessage(messages[`blockingReason.${blocking}`])
```
Comment (hellais) [inline components/measurement/nettests/WebConnectivity.js]: Yes this is good!

File: components/measurement/nettests/WebConnectivity.js [hunk 2]
```diff
@@ -338,13 +330,12 @@ const WebConnectivityDetails = ({
           WebsiteURL: input,
           network: probe_asn,
           country: country,
-          BlockingReason: <strong>{messages[`blockingReason.${blocking}`] && intl.formatMessage(messages[`blockingReason.${blocking}`])}</strong>
+          BlockingReason: reason && <strong>{reason}</strong>
```
Comment (hellais) [inline components/measurement/nettests/WebConnectivity.js]: Maybe we don't even have to do this `&&`, because worse case we create an empty `<strong></strong>` element.

File: pages/measurement.js [hunk 3]
```diff
@@ -50,7 +50,7 @@ export default class Measurement extends React.Component {
       initialProps['isAnomaly'] = results[0].anomaly
       initialProps['isFailure'] = results[0].failure
       initialProps['isConfirmed'] = results[0].confirmed
-
+      initialProps['message'] = results[0].scores.msg || null
```
Comment (hellais) [inline pages/measurement.js]: I think you should probably inject into the `measurement` the whole `scores` dictionary and we can maybe for the time being, for debugging purposes, just log it to the console.

Eventually I think we would like to show it somewhere in the measurement page itself, probably somewhere close to the heading.

The `scores` field is what you are going to be using in order to extract the WhatsApp related that are calculated by the fastpath, see: https://github.com/ooni/pipeline/pull/297/files#diff-88c24525382330f3fc745dfea1834e78R765.

Basically you will get a data structure like this:
```
"scores": {
      "analysis": {
        "registration_server_accessible": true,
        "whatsapp_web_accessible": false,
        "whatsapp_endpoints_accessible": true,
    }
}
```

Which map to what you have in the WhatsApp test as: https://github.com/ooni/explorer/blob/master/components/measurement/nettests/WhatsApp.js#L30

File: components/measurement/nettests/WebConnectivity.js (hunk 2 above)
Comment (sarathms) [reply components/measurement/nettests/WebConnectivity.js]: I generally try not to let unnecessary elements into the DOM. It has led layout issues in the past, although not exactly with this type of an element.

Comment (sarathms) [reply components/measurement/nettests/WebConnectivity.js]: That said, I don't claim that none have slipped past me :grin:

File: pages/measurement.js (hunk 3 above)
Comment (sarathms) [reply pages/measurement.js]: Done

**Comment by hellais** [comment]
This is the PR addressing: https://github.com/ooni/explorer/issues/403
